mu4/appshell/appshell.cpp

```diff
@@ -39,10 +41,23 @@ int AppShell::run(int argc, char** argv, std::function<void()> moduleSetup)
     qputenv("QT_STYLE_OVERRIDE", "Fusion");
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 
+    const char* appName;
+    if (framework::Version::unstable()) {
+        appName  = "MuseScore4Development";
```

Comment:
It seems we had the similliar cmake variables, maybe it's better to use them?

Reply:
In cmake a little different, I think now so be it (also in MU3)